src/core/maps/area_occupancy_estimator.h

```diff
@@ -61,9 +73,16 @@ class AreaOccupancyEstimator : public CellOccupancyEstimator {
 
 public: //methods
 
+  /*!
+   * Parameterized constructor sets all data members
+   * \param[in] occ, empty - base probabilities values of occupied (not occupied) cells (0.95 and 0.1 by default)
```

Comment:
"<...> probability values <...> occupied/not occupied cells." Don't mention the defaults here.